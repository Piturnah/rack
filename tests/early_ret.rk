// takes int, returns 0 if given 0x12
//            else ret 1
fn foo in
  0x12 = if
    0 ret
  end
  1
end

fn main in
  1 foo print
  0x12 foo print
end
