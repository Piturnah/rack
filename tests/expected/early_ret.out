----STDOUT----
1
0

----STDERR----
